### ISSUE TITLE
Create a package to quickly initialize an Asciidoctor PDF project

### ISSUE THREAD

**Issue by Mogztter** [Mogztter/asciidoctor-pdf.js#101]
Publish a package named `asciidoctor-pdf-project`. The idea is to quickly initialize a project using `npm init` command:

```
$ npm init asciidoctor-pdf-project ./my-asciidoctor-pdf-project
```

The package should contain:

- A sample document named `sample.adoc`
- A `package.json` file

Reference: https://docs.npmjs.com/cli/init

**Comment by oncletom** [comment]
The first example I read on https://docs.npmjs.com/cli/init is a bit misleading. What I understand is if you run `npm init asciidoctor-pdf-project`, the implied package name is `create-asciidoctor-pdf-project`.

An alias is `npx create-asciidoctor-pdf-project`.

**Comment by Mogztter** [comment]
Yes you're right, good catch 👍 
The package should be named `create-asciidoctor-pdf-project`. We can discuss about the name but it should start with `create-` by convention.

**Comment by oncletom** [comment]
Exactly. It sounds like `@asciidoctor/create-pdf-project` works too (depending on where you are up to regarding using the `@org` stuff.
